src/main/resources/application.yml

```diff
@@ -53,3 +53,6 @@ spring:
   rabbitmq:
     host: ${RABBITMQ_HOST}
     port: ${RABBITMQ_PORT}
+  jackson:
+    deserialization:
+      READ_UNKNOWN_ENUM_VALUES_AS_NULL: true
```

Comment:
I am not sure about the implications of this in other places of project. Had to add this due to validation, for the case of payload not containing a valid enum.

Reply:
If you want to accept the request as it's and do validation, I would request you to have two objects, 

one with string identifier type => which goes through the validation class comes out => one with enum identifier type => gets used in the java service classes.

Reply:
done